src/core/server/http/router/router.ts

```diff
@@ -141,9 +142,9 @@ function getRouteFullPath(routerPath: string, routePath: string) {
  * undefined.
  */
 function routeSchemasFromRouteConfig<
-  P extends ObjectType,
-  Q extends ObjectType,
-  B extends ObjectType | Type<Buffer> | Type<Stream>
+  P extends ObjectType | ValidateFunction<unknown>,
+  Q extends ObjectType | ValidateFunction<unknown>,
+  B extends ObjectType | Type<Buffer> | Type<Stream> | ValidateFunction<unknown>
```

Comment:
I think you forgot to change this a few lines after?

https://github.com/elastic/kibana/blob/066613e2a6b839ad893aa61fb3624187042b5bd3/src/core/server/http/router/router.ts#L156-L164

Reply:
Oh! I thought I did! Maybe it got lost when "fixing" the conflicts... Thank you for spotting it!